### ISSUE TITLE
Status update!

### ISSUE THREAD

**Issue by made-by-chris** [pelle/clothereum#1]
Hi! Is this still being developed?

**Comment by pelle** [comment]
@basiclaser No, not really. But I am working with clojure and Ethereum on a couple of projects. I will be Open Sourcing my internal clojurescript library soon. I also have plans of supporting regular Clojure using Ethereumj.

**Comment by made-by-chris** [comment]
Great! So are you finding clojurescript to be easier for developing with ethereum? If you don't mind me asking, what is your stack or strategy for working with ethereum? Are you targetting the problem space of writing contracts or something else? Thanks
